core/arch/arm/include/kernel/thread.h

```diff
@@ -605,6 +605,23 @@ struct mobj *thread_rpc_alloc_payload(size_t size);
  */
 void thread_rpc_free_payload(struct mobj *mobj);
 
+/**
+ * Allocates data for payload buffers only shared with the non secure kernel
```

Comment:
Pefer s/Allocates/Allocate/
s/non secure/non-secure/

Reply:
replacing allocates with allocate seems wrong to me but will do.

Reply:
Adding my two cents here: we have a mix of both forms currently, and TBH both seem OK to me as long as the description is proper English.
For the commit subjects however we insist on using the imperative mood, but it is a different topic.